lib/Unix/PID/Tiny.pm

```diff
@@ -242,9 +331,34 @@ Like Unix::PID but supplies only a few key functions.
 
 =head1 INTERFACE
 
-=head2 new()
+=head2 new(I<[$args_hr]>)
+
+See L<Unix::PID>'s new().  The following options can be provided in the
+optional HASH, I<$args_hr>, to enable certain extensions:
+
+=over
+
+=item B<keep_open>
```

Comment:
This property is only used w/ `close_on_destroy`. Why not simplify and drop `keep_open`?

Reply:
My second usage of close_on_destroy, wherein I would only satisfy the condition to not close the file handle if both were specified, was erroneous. This is fixed in commit 89499b5 now.